<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAccountDialog</class>
 <widget class="QDialog" name="AddAccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="accountFullNameLabel">
     <property name="text">
      <string>&amp;Full name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>accountFullNameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="statusText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="accountTypeLabel">
     <property name="text">
      <string>&amp;Account type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>accountTypeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="accountFullNameLineEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="accountTypeComboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="evernoteServerLabel">
     <property name="text">
      <string>&amp;Evernote server:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>evernoteServerComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="evernoteServerComboBox"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="accountUsernameLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="accountUsernameLabel">
     <property name="text">
      <string>&amp;Username</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>accountUsernameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="useNetworkProxyCheckBox">
     <property name="text">
      <string>Use network proxy</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QFrame" name="networkProxyFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="networkProxyTypeLabel">
        <property name="text">
         <string>Proxy &amp;type:</string>
        </property>
        <property name="buddy">
         <cstring>networkProxyTypeComboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="networkProxyTypeComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="networkProxyHostLabel">
        <property name="text">
         <string>&amp;Host:</string>
        </property>
        <property name="buddy">
         <cstring>networkProxyHostLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="networkProxyHostLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="networkProxyPortLabel">
        <property name="text">
         <string>&amp;Port:</string>
        </property>
        <property name="buddy">
         <cstring>networkProxyPortSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="networkProxyPortSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="networkProxyUserLabel">
        <property name="text">
         <string>&amp;User:</string>
        </property>
        <property name="buddy">
         <cstring>networkProxyUserLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="networkProxyUserLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="networkProxyPasswordLabel">
        <property name="text">
         <string>P&amp;assword:</string>
        </property>
        <property name="buddy">
         <cstring>networkProxyPasswordLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="networkProxyPasswordHorizontalLayout">
        <item>
         <widget class="QLineEdit" name="networkProxyPasswordLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="networkProxyPasswordShowCheckBox">
          <property name="text">
           <string>Show</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddAccountDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddAccountDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
